README.md

```diff
@@ -1,223 +1,104 @@
-   and navigate to the transformer directory `cd rudder-transformer`
-6. Install dependencies `npm i` and start the destination transformer `node destTransformer.js`
-7. Navigate back to main directory `cd rudder-server`. Copy the sample.env to the main directory `cp config/sample.env .env`
-8. Update the `WORKSPACE_TOKEN` environment variable with the token fetched in step 4
-9. Run the backend server `go run -mod=vendor main.go`
-10. Follow (Send Test Events) instructions below to send test event.
+RudderStack's backend is written in Go, with a rich UI written in React.js.
 
----
+You can also use the [cloud-hosted](https://app.rudderstack.com/signup?type=freetrial&utm_source=github&utm_medium=rdr-srv&utm_campaign=hosted&utm_content=intro) RudderStack instance to experience the product. Click [here](https://app.rudderstack.com/signup?type=freetrial&utm_source=github&utm_medium=rdr-srv&utm_campaign=hosted&utm_content=intro).
 
-# Send Test Events
+# Why Use RudderStack?
+- **Production-ready**: Companies like **MatterMost**, **IFTTT**, **Grofers**, and **1mg** use RudderStack for collecting their events
 
-1. If you already have a Google Analytics account, keep the tracking ID handy. If not, please create one and get the tracking ID. The Google Analytics account needs to have a **Web** Property (**Web+App** does't seem to work)
-2. Create one source (Android or iOS) and configure a Google Analytics destination for the same with the above tracking ID
-3. We have bundled a shell script that can generate test events. Get the source “writeKey” from our app dashboard and then run the following command. Run `cd scripts; ./generate-event <writeKeyHere> http://localhost:8080/v1/batch`. NOTE: `writeKey` is different from the `your_workspace_token` in step 2. Former is associated with the source while the latter is for your account.
-4. You can then login to your Google Analytics account and verify that events are delivered. Go to `MainPage->RealTime->Events`. `RealTime` view is important as the other dashboard can sometimes take 24-48 hrs to refresh.
-5. You can use our [Javascript][rudder-sdk-js-git-repo], [Android][rudder-sdk-android-git-repo] or [iOS][rudder-sdk-ios-git-repo] SDKs for sending events from your app.
+- **Extreme Scale**: One of our largest installations currently sends **300 Million** events/day with peak of **40K** req/sec, via a multi-node RudderStack setup
 
----
+- **Privacy and Security**: You can collect and store your customer data without sending everything to a third-party vendor. With RudderStack, you get fine-grained control over what data to forward to which analytical tool
 
-# RudderStack Config Generator
+- **Seamless Integration**: RudderStack currently supports integration with over 45 popular destination platforms such as Google Analytics, Amplitude, Mixpanel, Amazon S3,and more. Moreover, it also supports integration with popular data warehouses such as Snowflake, Redshift, and Google BigQuery
```

Comment:
Let's link to https://docs.rudderstack.com/destinations

Remove `such as` for warehouses. We only support 3 now.
Also link to https://docs.rudderstack.com/data-warehouse-integrations